### ISSUE TITLE
Always labels are tokenizing instead of text column, Kindly fix the issue facing 

### ISSUE THREAD

**Issue by suryapa1** [n-waves/multifit#75]
exp = multifit.from_pretrained("de_multifit_paper_version")
cls_dataset = exp.arch.dataset(Path('data/de_sentiment'), exp.pretrain_lm.tokenizer)
cls_dataset.load_clas_databunch(bs=exp.finetune_lm.bs).show_batch()

data/de_sentiment , path has train.csv/test.csv with labels, text as columns, even by shuffling as well show batch is tokenizing, Not sure why it is populaitng as such, any help is greatly apprecisted.,

My problem statement that is trying is as follows:
1) Get german pretrained  using multifit.from_pretrained("de_multifit_paper_version")
2) create custom classifer dataset and fine tune on top of german pretrained
3) classify custom dataset 

Any example is greatly appreciated as well,

**Comment by suryapa1** [comment]
Updated screenshot:
<img width="1013" alt="Screenshot 2020-06-15 at 11 02 29 PM" src="https://user-images.githubusercontent.com/6042186/84688156-7ef41980-af5c-11ea-8787-e7cf9b1b9718.png">
